tests/ot/test_assets.py

```diff
@@ -11206,3 +11208,14 @@ def test_asset(fixture_ot):
 
     resp = fixture_ot.assets.asset("fff9f1b9-26db-4bf4-9d87-bd915aaede36")
     assert resp.next() == expected
+
+
+@pytest.mark.vcr()
+def test_assets_list_vcr(api):
+    """
+    Tests the assets Graphql list iterator with cassettes
+    """
+    assets = api.assets.list(limit=2)
+    assert isinstance(assets, OTGraphIterator)
+    asset = assets.next()
+    assert asset is not None
```

Comment:
```suggestion
    assert asset is not None
    assert isinstance(asset, Asset)
```

Reply:
It is taken care of by the assert isinstance(assets, OTGraphIterator), which evaluates and returns list of assets.